src/main/java/util/FileUtil.java

```diff
@@ -0,0 +1,27 @@
+package util;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileUtil {
+
+    public List<String> getLineString(String pathFile) {
+        try {
+            return Files.readAllLines(Paths.get(pathFile), StandardCharsets.UTF_8);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file " + pathFile, e);
+        }
+    }
+
+    public void writeFile(String path, String report) {
+        try {
+            Files.writeString(Path.of(path), report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write file" + path + e);
+        }
+    }
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/183450140-1192aae8-2ee1-4be6-9bf4-504d0cd8b4e4.png)
create separate services to write and read files